fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -84,8 +84,13 @@
     // FIXITY
 
     public static final Resource FIXITY_TYPE = createResource(PREMIS_NAMESPACE + "Fixity");
+
+    public static final Property HAS_MESSAGE_DIGEST_ALGORITHM =
+            createProperty(PREMIS_NAMESPACE + "hasMessageDigestAlgorithm");
```

Comment:
I am not seeing the `hasMessageDigestAlgorithm` in the Premis namespace:
http://www.loc.gov/standards/premis/v1/premis-dd_1.0_2005_May.pdf